chef_master/source/aws_opsworks_chef_automate.rst

```diff
@@ -68,3 +68,96 @@ Enabling the compliance profile storage service in AWS OpsWorks for Chef Automat
 #. `Configure the audit cookbook <https://github.com/chef-cookbooks/audit#reporting-to-chef-visibility-via-chef-server>`_ to scan your nodes.
 
 .. note:: These instructions only detail what has to be added to the existing configuration as found in AWS OpsWorks for Chef Automate. For general instructions, see :doc:`integrate_compliance_chef_automate` (collector ``chef-server-visibility``) and :doc:`install_chef_automate`.
+
+Adding Push Jobs Server based build nodes to AWS OpsWorks for Chef Automate
+============================================================================
+
+Adding Push Jobs Server based build nodes to AWS OpsWorks for Chef Automate requires changes to the Chef Automate instance as well as the node that will run as a builder.
+
+#. Install Push Jobs Server on the Chef Automate instance by downloading the Push Jobs Server from https://downloads.chef.io/push-jobs-server and running:
```

Comment:
Is there a reason (accepting the license maybe?) we don't use `install.sh` for push jobs server?

Given that `curl -v "https://omnitruck-direct.chef.io/stable/push-jobs-server/metadata?v=&p=el&pv=7&m=x86_64"` returns

```
sha1    8c9351995b177a087a0fa4334f9acaee01643d8a
sha256  78a71e09d061e033c520bfd2c575b8d20eb9ebebc17b423954d3e5e7261dcf8c
url     https://packages.chef.io/files/stable/opscode-push-jobs-server/2.1.1/el/7/opscode-push-jobs-server-2.1.1-1.el7.x86_64.rpm
```
I'd think `| bash -s -- -c stable -P opscode-push-jobs-server` could do the trick.